src/query/react/versionedTypes/index.ts

```diff
@@ -5,4 +5,4 @@
 // Overall, this setup allows us to selectively override the one
 // file that has any difference between 4.1 and earlier, without
 // having to ship two completely duplicate copies of our typedefs.
-export { HooksWithUniqueNames } from './ts41Types'
+export type { HooksWithUniqueNames } from './ts41Types'
```

Comment:
I don't think this would be breaking, but we should check (@phryneas) - this was required for the vite example support, otherwise `isolatedModules` would have to change in the tsconfig.

```ts
r TS1205: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.

8 export { HooksWithUniqueNames } from './ts41Types'
           ~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```